### PR TITLE
MVC Code refactoring per talk with Luv

### PR DESCRIPTION
**Pull request by ryanybrennan**
login, logout, profile, registration, search_events, search_events_curlAPI moved to dynamic folder and corresponding actions moved to dynamic_controller. moved php code from login, registration, and logout to dynamic_controller as per MVC. Profile page and search_events still needs to be refactored regarding php code.

Team lead: Maria Contado
Team member: Ryan Brennan

### CODE REVIEW

**Comment by ryanybrennan** [comment]
Need to refactor current php code from straight SQL queries to more OOP code reflecting the backend classes still.

**Review by mviencek — CHANGES_REQUESTED** [review]
please pull dev and resolve a couple of conflicts

**Comment by ryanybrennan** [comment]
Apologies for the merge conflicts, must have missed an update.

File: controllers/static_controller.php [hunk 1]
```diff
@@ -9,35 +9,6 @@ public function landing() {
...
-
-    public function profile() {
-        require_once('views/static/profile.php');
-    }
-
-    public function logout() {
-        require_once('views/static/logout.php');
-    }
-
-    public function terms(){
```
Comment (mcontado) [inline controllers/static_controller.php:33]: I think the terms should stay in the static controller.

File: controllers/static_controller.php [hunk 2]
```diff
@@ -9,35 +9,6 @@ public function landing() {
...
-        require_once('views/static/terms.php');
-    }
-
-    public function contact() {
```
Comment (mcontado) [inline controllers/static_controller.php]: If you remove the contact here, please don't forget to add it in the dynamic controller. This new page got merged yesterday.

File: routes.php [hunk 3]
```diff
@@ -9,17 +9,15 @@ function call($controller, $action) {
...
             break;
     }
 
     $controller->{ $action }();
 }
 
-
-$controllers = array('static' => ['landing', 'error', 'registration', 'login', 'profile', 'logout', 'searchEvents', 'contact', 'terms'], 'dynamic' => ['home', 'error']);
-
+$controllers = array('static' => ['landing', 'error'], 'dynamic' => ['home', 'error', 'registration', 'login',  'searchEvents', 'logout', 'profile']);
```
Comment (mcontado) [inline routes.php]: please don't forget to add the contact and terms

**Review by mcontado — CHANGES_REQUESTED** [review]
Please don't forget to add the terms and contact pages in their respective controllers and update their php.

**Comment by mviencek** [comment]
@ryanybrennan  are you going to be able to make the requested changes tonight or tomorrow morning so we can get you merged?

**Comment by ryanybrennan** [comment]
I pushed some changes yesterday, forgot to add contact and terms back into the flow since I accidentally deleted them. Will update again once I get those fixed.

**Comment by ryanybrennan** [comment]
Will be ready tonight.

**Comment by ryanybrennan** [comment]
Made a new commit. Let me know if I need to change anything else. I pulled the most recent project before pushing, so I should have no merge conflicts.

**Review by mviencek — APPROVED** [review]

**Review by mcontado — APPROVED** [review]

**Pull request merged**